Automattic/jetpack
# Simple Payments

This introduces Simple Payments feature.

This is an "ecommerce light" feature.

# Stark team: we dont merge this. The Jetpack release lead does. Dont merge!

![zrzut ekranu 2017-07-24 o 19 30 44](https://user-images.githubusercontent.com/3775068/28536001-9e7ced24-70a6-11e7-8d04-fd4f378a67a2.png)

More details in projectpaymentbutton-p2

Warning: on calypso side, there still is work to do, but we are pretty certain this JP piece is production-ready.
Since there is no UI in JP, users will start using it when we relese it in calypso.

We are still tweaking CSS, otherwise we consider it ready

## Testing

1. Check this branch in your JP sandbox
1. Build calypso in dev
1. Go to post/page editor
1. Click insert -> payment button
1. Fill product details (email has to be a user email on PP sandbox infrastructure. You can use `artur.piszek+test2@automattic.com`)
1. This inserts the shortcode
1. Go to your page frontend
1. Do `localStorage.setItem('simple-payments-env', 'sandbox' );` to switch to Sandbox mode, refresh
1. Click "pay with paypal" to go with the flow. You can use my test sandbox account (`artur.piszek+test@automattic.com` , pass: `potato123`)
